lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2240,6 +2240,55 @@ describe('AwsCompileFunctions', () => {
         });
     });
 
+    it('should not create a new version object if only function-wide configuration changed', () => {
```

Comment:
All new tests should be written with help of `runServerless` util (see https://github.com/serverless/serverless/blob/master/test/README.md)

In this case I would rely on basic `function` fixture as follows

1. Setup a fixture with `fixture.setup`, as e.g. it's done here: https://github.com/serverless/serverless/blob/d9b91e97fb81b6f19c9f95920b509d623bdca37d/lib/utils/analytics/generatePayload.test.js#L18
2. Issue first `runServerless` and read all generated lambda version ids
3. Modify `reservedConcurrency` and `tags` with help of `updateConfig` (returned by `fixture.setup()`) as it's done here: https://github.com/serverless/serverless/blob/d9b91e97fb81b6f19c9f95920b509d623bdca37d/test/integration/apiGateway.test.js#L191-L204
4. Re-run `runServerless`, and confirm that version id didn't change

Reply:
Thanks @medikoo [I refactored the test](https://github.com/serverless/serverless/pull/8212/commits/8562f4f598675a2791fd444e5db8dbbdbd680227), but it seems to me the second call to `runServerless` isn't reading the updated config. I debugged the test and I did see `updateConfig` updating the `serverless.yml` in the temp dir, but as you can see in [this assertion (which fails)](https://github.com/thewizarodofoz/serverless/commit/8562f4f598675a2791fd444e5db8dbbdbd680227#diff-587fde9214d0bca41ecd7959b77058d6R2942), `runServerless` isn't picking up those changes.